9.2/test/run

```diff
@@ -212,6 +212,71 @@ function run_tests() {
   test_postgresql $name
 }
 
+function run_change_password_test() {
+  local name="change_password"
+
+  local database='db'
+  local user='user'
+  local password='password'
+  local admin_password='adminPassword'
+
+  local volume_dir
+  volume_dir=`mktemp -d --tmpdir pg-testdata.XXXXX`
+  chmod a+rwx ${volume_dir}
+
+  DOCKER_ARGS="
+-e POSTGRESQL_DATABASE=${database}
+-e POSTGRESQL_USER=${user}
+-e POSTGRESQL_PASSWORD=${password}
+-e POSTGRESQL_ADMIN_PASSWORD=${admin_password}
+-v ${volume_dir}:/var/lib/pgsql/data
+" create_container ${name}
+
+  # need to set these because `postgresql_cmd` relies on global variables
+  USER=${user}
+  PASS=${password}
+
+  # need this to wait for the container to start up
+  CONTAINER_IP=$(get_container_ip ${name})
+  test_connection ${name}
+
+  echo "  Testing login"
+  assert_login_access ${user} ${password} true
+  USER='postgres'
+  PASS=${admin_password}
+  assert_login_access 'postgres' ${admin_password} true
+
+  echo "  Changing passwords"
+  docker stop $(get_cid ${name})
+  DOCKER_ARGS="
+-e POSTGRESQL_DATABASE=${database}
+-e POSTGRESQL_USER=${user}
+-e POSTGRESQL_PASSWORD=NEW_${password}
+-e POSTGRESQL_ADMIN_PASSWORD=NEW_${admin_password}
+-v ${volume_dir}:/var/lib/pgsql/data
+" create_container "${name}_NEW"
+
+  # need to set these because `postgresql_cmd` relies on global variables
+  USER=${user}
+  PASS="NEW_${password}"
+
+  # need this to wait for the container to start up
+  CONTAINER_IP=$(get_container_ip "${name}_NEW")
+  test_connection "${name}_NEW"
+
+  echo "  Testing login with new passwords"
+  assert_login_access ${user} "NEW_${password}" true
+  USER='postgres'
+  PASS="NEW_${admin_password}"
+  assert_login_access 'postgres' "NEW_${admin_password}" true
```

Comment:
I'd say in general if one password works, the other, by exclusion, doesn't. But in this reality we have here testing both can make sense...

I'm almost sure `assert_login_access` always return 0, though. So all those tests always pass...